tests/conftest.py

```diff
@@ -66,5 +66,5 @@ def login_link(username):
     mail = restmail.get_mail(username)
     mail_content = mail[0]['text'].replace('\n', ' ').replace('amp;', '').split(" ")
     for link in mail_content:
-        if link.startswith("https"):
+        if link.startswith("https://auth.mozilla.auth0"):
```

Comment:
I suggest you add a larger portion of the URL to make sure that we don't have false alarms if for some reason there is another similar link. eg

``link.startswith("https://auth.mozilla.auth0.com/passwordless/verify_redirect"``